glib/src/gstring.rs

```diff
@@ -537,6 +542,10 @@ impl<'a> ToGlibPtr<'a, *mut c_char> for GStr {
     }
 }
 
+// size_of::<Inner>() minus two bytes for length and enum discriminant
+const INLINE_LEN: usize =
+    mem::size_of::<Option<Box<str>>>() + mem::size_of::<usize>() - mem::size_of::<u8>() * 2;
```

Comment:
Can we get a test for making sure this doesn't unnecessarily increase the size of `GString`?

Reply:
Added layout test